crates/mentat-types/src/identifiers/account_identifier.rs

```diff
@@ -1,5 +1,7 @@
 //! The module defines the `AccountIdentifier`.
 
+use std::mem::size_of_val;
```

Comment:
NIT: we shouldn't need to use IndexMap since it should already be in super::*, I believe? size_of_val can be imported from super as well to avoid repetitive imports.

Reply:
the indexmaps were actually there before this pr. but yeah ill clean em up